sonarts-sq-plugin/its/plugin/src/test/java/org/sonar/typescript/its/TslintRulesTest.java

```diff
@@ -39,19 +40,29 @@
   public static Orchestrator orchestrator = Tests.ORCHESTRATOR;
 
   @Test
-  public void should_avoid_escape_in_quote_rule() {
-    assertIssueLines("S1441", "profile-single-quotes", ImmutableList.of(1));
-    assertIssueLines("S1441", "profile-double-quotes", ImmutableList.of(3));
+  public void S1441_should_avoid_escape_in_quote_rule() {
+    List<Issue> issueList = getIssues("S1441", "profile-single-quotes");
+    assertThat(issueList).extracting("line").containsExactlyElementsOf(ImmutableList.of(1));
+
+    issueList = getIssues("S1441", "profile-double-quotes");
+    assertThat(issueList).extracting("line").containsExactlyElementsOf(ImmutableList.of(3));
+  }
+
+  @Test
+  public void S4328_should_consider_peer_dependencies() {
+    List<Issue> issueList = getIssues("S4328", "profile-no-implicit-dependencies");
+    assertThat(issueList).extracting("line").doesNotContainAnyElementsOf(ImmutableList.of(1));
```

Comment:
I'm not sure to understand. Shouldn't the list be purely empty? I think that would be more clear as test, except than saying "everything but not line one"